config/initializers/elasticsearch.rb

```diff
@@ -1,14 +1,12 @@
 # frozen_string_literal: true
 
-Elasticsearch::Persistence.client = Elasticsearch::Client.new(
-  log: Rails.configuration.elasticsearch['log'],
-  hosts: Rails.configuration.elasticsearch['hosts'],
-  user: Rails.configuration.elasticsearch['user'],
-  password: Rails.configuration.elasticsearch['password'],
-  randomize_hosts: true,
-  retry_on_failure: true,
-  reload_connections: true
-)
+ES_CONFIG = Rails.application.config_for(:elasticsearch).freeze
+
+def config
```

Comment:
Can you help me understand what this is doing?  It seems empty and not needed.

Reply:
Ah, sorry about that. I meant to remove this. Will push up a new commit shortly.